pages/common/blackfire.md

```diff
@@ -0,0 +1,36 @@
+# blackfire
+
+> A command line profiling tool for PHP.
+> Homepage: <https://blackfire.io>.
+
+- Initialise and configure the Blackfire client:
+
+`blackfire config`
+
+- Launch the Blackfire agent:
+
+`blackfire agent`
+
+- Launch the Blackfire agent on a specific socket:
+
+`blackfire agent --socket="{{tcp://127.0.0.1:8307}}"`
+
+- Run the profiler on a specific program:
+
+`blackfire run {{php path/to/file.php}}`
+
+- Run the profiler and collect 10 samples:
+
+`blackfire --samples={{10}} run {{php path/to/file.php}}`
+
+- Run the profiler and output results as JSON:
+
+`blackfire --json run {{php path/to/file.php}}`
+
+- Upload a profiler file:
+
+`blackfire upload {{path/to/file}}`
+
+- View the status of profiles:
```

Comment:
How about "profiles" → "profiler file", to match the language of the previous example? Or is this a different concept?